stock_available/models/product_product.py

```diff
@@ -36,8 +36,10 @@ def _immediately_usable_qty(self):
 
         By default, available to promise = forecasted quantity.
 
-        Must be overridden by another module that actually implement
-        computations."""
+        EACH sub-module MUST override this method in BOTH product.product
```

Comment:
I will. I always forget RST is a richer than plain text, sorry. To much 
reading RFCs in my young time probably.